modules/apps/frontend-theme/frontend-theme-classic/src/css/components/_navbars.scss

```diff
@@ -1,50 +0,0 @@
-.navbar-classic {
-	background-color: #fff;
-
-	&.navbar-light {
-		border-bottom: 1px solid $light;
-	}
-
-	.navbar-form {
-		padding: 0;
-
-		.field-wrapper {
-			right: 0;
-			top: 0;
-
-			.lfr-icon-item {
-				color: #6b6c7e;
-			}
-		}
-	}
-
-	.portlet-search-bar {
-		@include media-breakpoint-down(sm) {
-			width: 100%;
-		}
-
-		.input-group-item {
-			margin-left: 0;
-		}
-	}
-
-	.logo:hover {
-		text-decoration: none;
-	}
-
-	.navbar-nav {
-		.nav-item:not(:last-of-type) {
-			.nav-link {
-				@include media-breakpoint-up(lg) {
-					margin-right: 16px;
```

Comment:
Removing this I've changed the space between the nav-links, but do we need that?

![last-link-1](https://user-images.githubusercontent.com/46778114/107676092-0be0a580-6c99-11eb-8c91-cdf3d17932b2.jpg)
![last-link-2](https://user-images.githubusercontent.com/46778114/107676124-14d17700-6c99-11eb-8bfe-2746936f4cd6.jpg)